### ISSUE TITLE
Material – Viewing Forum List instead of View List of Online Members on Members Activity

### ISSUE THREAD

**Issue by dprock** [Audentio/xftheme-issues#167]
[Solidus](https://xenforo.com/community/members/solidus.15716/) wrote: ([view on xenforo.com](https://xenforo.com/community/threads/material-paid.125297/page-11#post-1161983))
> [@ThemeHouse](https://xenforo.com/community/members/83700/) [@Dalton Prock](https://xenforo.com/community/members/14408/) very strange bug here. When using this theme, all members activity shows as "Viewing Forum List" even when they aren't. Here is me viewing list of online members,  
>   
> [1.png]  
>   
> and now in the default theme  
> [2.png]  
>   
> This doesn't happen on Abyss either.

**Comment by nsaneman** [comment]
@dprock This seems like an issue caused by a plugin because I see the correct information with both the default XenForo theme and the Material theme.

If I use **any** theme and click to view who's online then I'll see myself listed as "Viewing forum list". Only when I refresh the page does it change to "Viewing list of online members". Everyone else is either viewing threads, forums or they're actually viewing the forum list.

**Comment by mgsolidus** [comment]
I can reproduce this on the ThemeHouse demo site as well.
Here I am logged in on the demo account refreshing the "Current Visitors" page,

Material: https://gfxf.net/images/2017/07/22/bug.gif
Tactical Light: https://gfxf.net/images/2017/07/22/notabug.gif

This is a guests view of the Demo account when Demo is using...

Material: https://gfxf.net/images/2017/07/22/1.png
UIX: https://gfxf.net/image/EmN

**Comment by nsaneman** [comment]
@mgsolidus You're right. With the default Material theme I can replicate this, but with my modified child I don't have this issue at all.

Search for the property "metroTitlebarImage" and disable it. Problem fixed.

**Comment by mgsolidus** [comment]
metroTitlebarImage is too nice to disable, I'll just deal with it.

**Comment by mikecreuzer** [comment]
Yeah, I've told Ian we should hit Material one more time to really polish it off. Only caveat is I'm not sure when we can get to it, but we definitely will do so as soon as UI.X 2 is a little bit closer to completion. We're at the 90% ish point.
